src/CloudEvents/Serializers/Formatters/Formatter.php

```diff
@@ -27,9 +27,11 @@ public function encodeTime(?DateTimeInterface $time): ?string
 
     public function decodeTime(?string $time): ?DateTimeInterface
     {
-        return $time === null
+        $parsed = DateTimeImmutable::createFromFormat(self::TIME_FORMAT, $time, new DateTimeZone(self::TIME_ZONE));
```

Comment:
This code will break on PHP 7.1 because it is strict about passing null to a string parameter.